zilliqa/src/db.rs

```diff
@@ -226,10 +231,8 @@ impl Db {
                 gas_used INTEGER NOT NULL,
                 gas_limit INTEGER NOT NULL,
                 qc BLOB NOT NULL,
-                agg BLOB);
-            CREATE TABLE IF NOT EXISTS main_chain_canonical_blocks (
-                height INTEGER NOT NULL PRIMARY KEY,
-                block_hash TEXT NOT NULL REFERENCES blocks (block_hash));
+                agg BLOB,
+                is_canonical BOOLEAN NOT NULL);
```

Comment:
Don't merge this yet - It makes a backwards-incompatible change to the database which we don't have good support for yet (e.g. DB migrations). We should discuss how to deal with this.